src/Uno.UI/UI/Xaml/DependencyPropertyDetailsCollection.Bindings.cs

```diff
@@ -219,5 +226,27 @@ internal BindingExpression GetBindingExpression(DependencyProperty dependencyPro
 
 			return null;
 		}
+
+		class BindingExpressionComparer : IComparer<BindingExpression>
```

Comment:
This should be singleton.
Also, specify the default access modifier (`private`) and make it sealed as well.